opentracing-impl/src/main/java/io/opentracing/propagation/TextFormatInjector.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2016 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing.propagation;
+
+import java.util.Map;
+
+/**
+ *
+ */
+interface TextFormatInjector extends Injector<Map> {
```

Comment:
@bensigelman, @yurishkuro,  we had a discussion recently about `Map` vs `TextCarrier` and the benefits of typing explicitly the required formats. Where did we land on that? (where was that discussion?)